web_interface/blueprints/api_v3.py

```
@@ -6427,7 +6431,8 @@ def get_logs():
     try:
         # Get recent logs from journalctl
         result = subprocess.run(
-            ['sudo', 'journalctl', '-u', 'ledmatrix.service', '-n', '100', '--no-pager'],
+            ['sudo', 'journalctl', '-u', 'ledmatrix.service', '-u', 'ledmatrix-web.service',
+             '-n', '100', '--no-pager', '--output=short-iso'],
```

Ruff (0.15.14)

[error] Starting a process with a partial executable path

(S607)